src/tike/ptycho/ptycho.py

```diff
@@ -171,20 +171,20 @@ def simulate(
 
 
 def reconstruct(
-        data,
-        probe, scan,
-        algorithm=None,
-        psi=None, num_gpu=1, num_iter=1, rtol=-1,
-        model='gaussian', use_mpi=False,
-        costs=[], times=[],
-        eigen_probe=None, eigen_weights=None,
-        batch_size=None,
-        initial_scan=None,
-        position_options=None,
-        probe_options=None,
-        object_options=None,
-        **kwargs
-):  # yapf: disable
+    data,
+    probe,
+    scan,
+    algorithm_options=solvers.EpieOptions(),
+    eigen_probe=None,
+    eigen_weights=None,
+    model='gaussian',
+    num_gpu=1,
+    object_options=None,
+    position_options=None,
+    probe_options=None,
+    psi=None,
+    use_mpi=False,
```

Comment:
These parameters are actually parameters of the `Operator` and `Communicator` objects, I've been considering having the user construct these directly instead of doing it internally, but I just haven't gotten around to it yet because I think it requires some refactoring of the Operator classes to make them less stateful and more functional.